packages/story-editor/src/components/publishModal/stories/index.js

```diff
@@ -96,7 +97,9 @@ export const _default = (args) => {
             },
           }}
         >
-          <PublishModal {...args} />
+          <ChecklistCountProvider hasChecklist={args.hasChecklist}>
```

Comment:
args is getting destructured in #10585 so this is mostly to prevent a conflict, i'll add it to the destructure in whatever PR is merged last between these two